src/js/elements/data_mapper/indicator.js

```diff
@@ -0,0 +1,137 @@
+import {Component} from "../../utils";
+import {SubIndicator} from "./subindicator";
+
+export class Indicator extends Component {
+    constructor(parent,
+                text,
+                id,
+                choroplethMethod,
+                primaryGroup,
+                versionData,
+                metadata,
+                choroplethRange,
+                enableLinearScrubber,
+                chartConfiguration) {
+        super(parent);
+
+        this._element = $($('.styles .data-category__h2')[1])
+            .find('.data-category__h3')[0].cloneNode(true);  //[0] -> v1  ---  [1] -> v2
```

Comment:
So `.find('.data-category__h3')[1]` would be v2
It's not perfect that the selector works that way but I think it's okay with the comment

Reply:
👍